zk

```diff
@@ -310,6 +311,16 @@ function open_browser() {
   $ZETTELKASTEN_BROWSER_COMMAND "https://github.com/$github_endpoint/tree/master/$ZETTELKASTEN_GIT_DIR/$zettel"
 }
 
+function push() {
+    cwd=$(pwd)
+    cd "$ZETTELKASTEN_DIR"
+    msg="Regenerated at $(date -u '+%Y-%m-%d %H:%M:%S') UTC"
```

Comment:
I think using the `get_datestamp()` function would be better here. What do you think?
https://github.com/AndrewCopeland/zettelkasten/blob/master/zk#L204-L211

Reply:
Yeah that should work fine, didn't notice that function there!